### PR TITLE
chore: remove ee direct imports

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

Replaces ee direct imports by strapi.ee. I'm keeping the old strapi.EE as is for now but will probably refactor later to merged into a single EE thing. 

Side note: as we are considering feature flag we could find a unified way to manage features that are enabled/disabled between flags & ee potentialy cc @Feranchz =)

### Why is it needed?

Removing dependencies to `@strapi/strapi`

### How to test it?

Verify the EE features are still toggled & working fine 

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/core/admin/_internal/node/build.ts [hunk 1]
```diff
@@ -79,8 +79,6 @@ const build = async ({ logger, cwd, tsconfig, ignorePrompts, ...options }: Build
   console.log('');
 
   try {
-    EE.init(cwd);
```
Comment (Marc-Roig) [inline packages/core/admin/_internal/node/build.ts:82]: where was the init moved to ? Is it not necessary to initialize it?

Comment (alexandrebodin) [reply packages/core/admin/_internal/node/build.ts:82]: It was usefull before to know at build time the EE feature for the FE but nowdays it's managed at runtime so we don't need that anymore. I couldn't fine any other reason to keep it but we will need to test that during our alpha/beta for sure :)

File: packages/core/admin/_internal/node/build.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { writeStaticClientFiles } from './staticFiles';
 import { build as buildWebpack } from './webpack/build';
 import { createBuildContext } from './createBuildContext';
 
-import EE from '@strapi/strapi/dist/utils/ee';
+// import EE from '@strapi/strapi/dist/utils/ee';
```
Comment (joshuaellis) [inline packages/core/admin/_internal/node/build.ts:7]: rogue comment

File: packages/core/admin/_internal/node/develop.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ import { createBuildContext } from './createBuildContext';
 import { WebpackWatcher, watch as watchWebpack } from './webpack/watch';
 import { build as buildWebpack } from './webpack/build';
 
-import EE from '@strapi/strapi/dist/utils/ee';
+// import EE from '@strapi/strapi/dist/utils/ee';
```
Comment (joshuaellis) [inline packages/core/admin/_internal/node/develop.ts:13]: rogue comment

**Review by Marc-Roig — APPROVED** [review]

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**